File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/SafeTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class SafeTag extends Tag {
    public SafeTag(int count) {
        super(count, Constants.SAFE_TAG, Constants.SAFE_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/SafeTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class SafeTag(count: Int) : Tag(count, Constants.SAFE_TAG, Constants.SAFE_TAG_REMOTE, HashSet(listOf(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))